### ISSUE TITLE
add a feature to modifiy the viewmodel bobbing and have portal 1 viewmodel bobbing

### ISSUE THREAD

**Issue by kiddu25** [StrataSource/Portal-2-Community-Edition#1748]
### Describe the bug

for show what I want to say I have 2 video 

https://www.youtube.com/watch?v=Mj6H9x5yHOI&ab_channel=kiddumaisl%27autre (portal 1 compatibility)
https://www.youtube.com/watch?v=BWxejBSzSIA change the bobbing (it's a portal 1 mapbase actually in developpement (not mine) leak code sadly but is only way for  show what I want )

### To Reproduce

1. open any map in portal 1/p2ce

### Operating System

not important

**Comment by vrad-exe** [comment]
This is a feature request, not a bug?

**Comment by kiddu25** [comment]
OzxyBox said to me to put that, so I choose that

> This is a feature request, not a bug?

**Comment by ozxybox** [comment]
yeah i forgot to reclassify it as a feature haha. it's been fixed now though. 
we might want to add a new template option for compat features, but oh well
